### ISSUE TITLE
make it easy to embed it in the existing plugin similar to vital.vim

### ISSUE THREAD

**Issue by prabirshrestha** [m-pilia/vim-yggdrasil#5]
```
:Vitalize --name=your_plugin_name $HOME/.vim/bundle/your_plugin_dir/
```

**Comment by m-pilia** [comment]
I had been thinking about this. Vital is a big dependency and poses contraints both on this plugin and the on the user.

If the plugin is implemented only with autoload scripts it should be possible to embed it easily without external tools, with a setup that copies the `autoload` folder in a sub-folder of the user plugin (e.g. `some_plugin/autoload/my_plugin/my_tree`), and then runs something like
```
find some_plugin/autoload/my_plugin/my_tree \
    -name '*.vim' \
    -exec sed -i 's,\byggdrasil#\b,my_plugin#my_tree#,' {}
```

**Comment by m-pilia** [comment]
Resolved in #8
